sale_triple_discount/views/sale_order_view.xml

```diff
@@ -12,6 +12,8 @@
                        groups="sale.group_discount_per_so_line"/>
                 <field name="discount3"
                        groups="sale.group_discount_per_so_line"/>
+                <field name="discounting_type"
+                       groups="sale.group_discount_per_so_line"/>
```

Comment:
You should add it to the form view as well (some lines below)

Reply:
Then it's only needed in the view, not on each field.

Reply:
Attended :wink: